deploy-apps/rolling-deploy.html.md.erb

```diff
@@ -1,10 +1,10 @@
 ---
-title: Configuring rolling app deployments
+title: Configuring app deployments
 owner: CAPI
 ---
 
 You can use Cloud Foundry Command Line Interface (cf CLI) commands
```

Comment:
*thought* @pspinrad Is it possible to get the existing links of like https://docs.cloudfoundry.org/devguide/deploy-apps/canary-deploy.html and https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html to redirect to this new combined page?